public/index.html

```diff
@@ -29,8 +29,8 @@ <h2>Loading, hold on</h2>
         </div>
         <div style="flex-grow: 1;"></div>
     </div>
-    <link href="https://fonts.googleapis.com/css?family=Muli:400,600,700" rel="stylesheet">
-    <link href="https://fonts.googleapis.com/css?family=Fira+Mono&text=0123456789ABCDEFGHJKLMNPQRSTUVXY" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Muli:400,600,700" rel="preload" as="style" onload="this.rel='stylesheet'">
+    <link href="https://fonts.googleapis.com/css?family=Fira+Mono&text=0123456789ABCDEFGHJKLMNPQRSTUVXY" rel="preload" as="style" onload="this.rel='stylesheet'">
```

Comment:
Cool but probably not CSP conform.
Also will trigger the download quite late, i.e. when the browser is idle and probably after the other preloads that webpack injects into the head.

Reply:
Webpack is using `prefetch`. `preload` downloads directly, I think.